### DISCUSSION TITLE
Host managment

### DISCUSSION THREAD

**Discussion by vudex** [kanidm/kanidm#1811] [Q&A] [answered]
As I understood from the FAQ, kanidm does not implement host managment in a way FreeIPA does. I mean HBAC and Sudo rules etc. Is it so or maybe I misread? Are there any plans for this?

**Comment by yaleman** [comment, marked as answer]
That's correct, it's not available at the moment.

In the future, it's definitely on the list - but there's a long list of features to build - we'll happily accept help!

**Reply by andreas-bulling** [reply]
Since the original answer is nearly three years old - I wondered whether these features (HBAC, sudo, etc) have been implemented by now?

**Reply by Firstyear** [reply]
They are still wanted but we have been focused on other areas of the project.
